suite-native/assets/src/components/AssetItem.tsx

```diff
@@ -72,6 +75,17 @@ export const AssetItem = memo(
             ) => selectNumberOfUniqueTokensForCoinPerDevice(state, cryptoCurrencySymbol),
         );
 
+        const accountsWithStaking = useSelector(
+            (
+                state: AccountsRootState &
+                    DeviceRootState &
+                    FiatRatesRootState &
+                    SettingsSliceRootState,
+            ) => selectVisibleDeviceAccountsWithStakingByNetworkSymbol(state, cryptoCurrencySymbol),
+        );
+
+        const isStaking = accountsWithStaking.length > 0;
```

Comment:
Name should be something like `hasAnyAccountWithStaking`

Reply:
https://github.com/trezor/trezor-suite/pull/14072/commits/565eadf4d973e9eded8dc5caa0e6da4d7dc37733